packages/mobile/src/components/premium-track-purchase-drawer/PremiumTrackPurchaseDrawer.tsx

```diff
@@ -200,6 +208,10 @@ const RenderForm = ({ track }: { track: PurchasableTrackMetadata }) => {
   const styles = useStyles()
   const { specialLightGreen, secondary } = useThemeColors()
   const presetValues = usePayExtraPresets(useRemoteVar)
+  const { isEnabled: isIOSUSDCPurchaseEnabled } = useFeatureFlag(
+    FeatureFlags.IOS_USDC_PURCHASE_ENABLED
+  )
+  const isIOSDisabled = Platform.OS === 'ios' && !isIOSUSDCPurchaseEnabled
```

Comment:
i think we have an isIOS helper somewhere

Reply:
hm couldn't find one, all i found was this: https://github.com/AudiusProject/audius-protocol/blob/a5bb8e3af9fb7b2694349347c4210efa140e71ce/packages/mobile/src/screens/settings-screen/SettingsScreen.tsx#L29

Reply:
sgsg